### ISSUE TITLE
Simple manager for Mac

### ISSUE THREAD

**Issue by Dannnyho** [r2-studio/robotmon-desktop#102]
May i know if simple manger 1.5 have a Mac verison?

**Comment by poi5305** [comment]
Yes. https://github.com/r2-studio/robotmon-desktop/releases/tag/v1.5

**Comment by Dannnyho** [comment]
> Yes. https://github.com/r2-studio/robotmon-desktop/releases/tag/v1.5
which file i can use for mac??

**Comment by poi5305** [comment]
simple-manager-v1-5-drawin

**Comment by hurryhenry** [comment]
Is there a simple manager for M1 Mac?
